server-openid4vci/src/main/java/com/android/identity/server/openid4vci/AuthorizeServlet.kt

```diff
@@ -16,56 +40,147 @@ import kotlin.time.Duration.Companion.minutes
  * redirect_uri supplied to [ParServlet] on the previous step.
  */
 class AuthorizeServlet : BaseServlet() {
-
+    companion object {
+        const val RESOURCE_BASE = "openid4vci"
+    }
     /**
      * Create a simple web page for the user to enter their name.
      *
      * TODO: flesh this out, use web API for PID-based authentication.
```

Comment:
Has this TODO been addressed, or is there more to do?

And does the function description also need to be updated? We aren't asking for the user's name anymore.